extensions/amp-sidebar/0.1/amp-sidebar.md

```diff
@@ -326,6 +326,49 @@ When using `toolbar` feature, `autoscroll` only works if the `<nav toolbar>` ele
 
 Please see [this example file](https://github.com/ampproject/amphtml/blob/master/examples/amp-sidebar-autoscroll.amp.html) for a working example code.
 
+## Building Nested Menus
+
+`<amp-sidebar>` can supports drilldown (nested) menus through a child component named [`<amp-nested-menu>`](https://amp.dev/documentation/components/amp-nested-menu/). With `<amp-nested-menu>`, `<amp-sidebar>` can support nesting one or more layers of submenus (and transition between them) as demonstrated by the following example:
+
+[tip type="note"]
+We recommend wrapping every menu item in a `li` element to improve accessibility and keyboard support.
+[/tip]
+
+[example playground="true" preview="top-frame" imports="amp-sidebar"]
+
+```html
+<button on="tap:sidebar1">Open Sidebar</button>
+<amp-sidebar id="sidebar1" layout="nodisplay" style="width:300px">
+  <amp-nested-menu layout="fill">
+    <ul>
+      <li>
+        <h4 amp-nested-submenu-open>Open Sub-Menu</h4>
+        <div amp-nested-submenu>
+          <ul>
+            <li>
+              <h4 amp-nested-submenu-close>Go back</h4>
+            </li>
+            <li>
+              <h4 amp-nested-submenu-open>Open Another Sub-Menu</h4>
+              <div amp-nested-submenu>
+                <h4 amp-nested-submenu-close>Go back</h4>
+              </div>
+            </li>
+          </ul>
+        </div>
+      </li>
+      <li>
+        <a href="https://amp.dev/">Link</a>
+      </li>
+    </ul>
+  </amp-nested-menu>
+</amp-sidebar>
+```
+
+[/example]
+
+See [`<amp-nested-menu>`](https://amp.dev/documentation/components/amp-nested-menu/) for the full documentation on nested menus along with advanced feature such as dynamic content loading.
```

Comment:
Is `amp-nested-menu` still experimental? We should delete `experimental` from the documentation when we remove the experiment flag.